mapows.h

```diff
@@ -149,7 +149,10 @@ MS_DLL_EXPORT const char * msOWSLookupMetadata2(hashTableObj *pri,
     hashTableObj *sec,
     const char *namespaces,
     const char *name);
-    
+
+MS_DLL_EXPORT int updateGMLFieldMetadata(layerObj *layer, const char *field_name, const char *gml_type, const char *gml_width, 
```

Comment:
no need to export this
```suggestion
int updateGMLFieldMetadata(layerObj *layer, const char *field_name, const char *gml_type, const char *gml_width, 
```

Reply:
@rouault - I tried this, but as the MSSQL is a separate plugin/DLL I think this is required https://ci.appveyor.com/project/MapServer/mapserver/builds/35196629/job/t24q8bmyf2kc0kob#L428

`mapmssql2008.obj : error LNK2019: unresolved external symbol _updateGMLFieldMetadata referenced in function _columnName `

Reply:
ah, good point. Could you then rename the function as ```msUpdateGMLFieldMetadata``` ?

Reply:
Done